packages/hardhat/test/Challenge0.ts

```diff
@@ -19,18 +19,21 @@ describe("🚩 Challenge 0: 🎟 Simple NFT Example 🤓", function () {
 
   describe("YourCollectible", function () {
     const contractAddress = process.env.CONTRACT_ADDRESS;
+
+    let contractArtifact: string;
     if (contractAddress) {
-      it("Should connect to external contract", async function () {
-        myContract = await ethers.getContractAt("YourCollectible", contractAddress);
-        console.log("     🛰 Connected to external contract", myContract.address);
-      });
+      // For the autograder.
+      contractArtifact = `contracts/download-${contractAddress}.sol:YourCollectible`
     } else {
-      it("Should deploy YourCollectible", async function () {
-        const YourCollectible = await ethers.getContractFactory("YourCollectible");
-        myContract = await YourCollectible.deploy();
-      });
+      contractArtifact = "contracts/YourCollectible.sol:YourCollectible";
     }
```

Comment:
I think the reason we did it this way nd not going the `let contractArtifact: string;` route is : 

If you run `CONTRACT_ADDRESS=0x2a32333141E12DF1883A15537683A4bda72b9D2c yarn test --network sepolia`

This code will connect to that address on Sepolia (Instead of deploying a new copy of the contract) and then do other transactions on that deployed adderess, while in `let contractArtifact: string;` will deploy new contrat and do transaction on that new deployed instance. 

Lol both approches make sense to me but I think going `let contractArtifact: string;` (this PR changes) will make our life easy too